plugins/node/opentelemetry-instrumentation-mongodb/src/instrumentation.ts

```diff
@@ -153,12 +154,13 @@ export class MongoDBInstrumentation extends InstrumentationBase<
             kind: SpanKind.CLIENT,
           }
         );
+
         instrumentation._populateAttributes(
           span,
           ns,
           server,
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          operationName !== 'insert' ? (ops[0] as any) : undefined
```

Comment:
I'm wondering why the previous version was like this. Are we sure we can remove this without breaking anything?

Reply:
I looked up why this is like this and it seems i introduced it when porting to instrumentation but i don't remember why. We only support one version range (>3.3 <4) and we are testing with 3.6
If tests are green i guess it was just a left over from trying to support the version 2.x (which i dropped afterwards while porting)